src/view/scripts.inc

```diff
@@ -0,0 +1,48 @@
+<!-- Namespace for Skosmos variables -->
+<script>
+  let SKOSMOS = (function () {
+    var content_lang = "{{ request.contentLang }}";
+    var explicitLangCodes = {{ explicit_langcodes ? 'true' : 'false' }};
+    var lang = "{{ request.lang }}";
+    var vocab = "{{ request.vocabid }}";
+    var waypoint_results = {{ parameters ? parameters.searchLimit : NULL }};
+
+    {% if request.page == 'page' and search_results and search_results|length == 1 %}
+      var prefLabels = [{"lang": "{{ search_results|first.label.lang }}","label": "{{ search_results|first.label }}"}{% for lang in search_results|first.foreignLabels %}{% for literal in lang %}{% if literal.type == 'skos:prefLabel' %},{"lang": "{{literal.lang}}", "label": "{{literal.label}}"}{% endif %}{% endfor %}{% endfor %}];
```

Comment:
There's quite a bit of spaghetti code in this. It's not clear to me what the code is actually doing. I wonder if we could move the logic to the PHP side? (possibly out of scope for this PR though)

Reply:
Leaving for later.